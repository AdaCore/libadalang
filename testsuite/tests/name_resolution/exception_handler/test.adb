procedure Test is
   package Pkg is
      Error : exception;
   end Pkg;

   procedure Foo is
   begin
   exception
      when Pkg.Error =>
         null;
   end Test;
   pragma Test_Block;
begin
   null;
end Test;
