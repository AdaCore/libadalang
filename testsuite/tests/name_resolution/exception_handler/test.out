Analyzing test.adb
##################

Resolving xrefs for node <SubpSpec test.adb:6:4-6:17>
*****************************************************


Resolving xrefs for node <ExceptionHandler [] test.adb:9:7-10:15>
*****************************************************************

Expr: <DottedName test.adb:9:12-9:21>
  references:    <DefiningName "Error" test.adb:3:7-3:12>
  type:          None
  expected type: None
Expr: <Id "Pkg" test.adb:9:12-9:15>
  references:    <DefiningName "Pkg" test.adb:2:12-2:15>
  type:          None
  expected type: None
Expr: <Id "Error" test.adb:9:16-9:21>
  references:    <DefiningName "Error" test.adb:3:7-3:12>
  type:          None
  expected type: None

Resolving xrefs for node <NullStmt test.adb:10:10-10:15>
********************************************************


Resolving xrefs for node <EndName test.adb:11:8-11:12>
******************************************************

Expr: <EndName test.adb:11:8-11:12>
  references:    <DefiningName "Foo" test.adb:6:14-6:17>
  type:          None
  expected type: None
Expr: <Id "Test" test.adb:11:8-11:12>
  references:    <DefiningName "Foo" test.adb:6:14-6:17>
  type:          None
  expected type: None


Done.
